python/sglang/srt/disaggregation/conn.py

```diff
@@ -315,21 +383,28 @@ def failure_exception(self):
 
 
 class KVReceiver:
-
     def __init__(
         self, mgr: KVManager, bootstrap_addr: str, bootstrap_room: Optional[int] = None
     ):
         self.bootstrap_room = bootstrap_room
         self.bootstrap_addr = bootstrap_addr
         self.kv_mgr = mgr
-        self.prefill_server_url = (
-            bootstrap_addr.split(":")[0]
-            + ":"
-            + str(KVSENDER_POLLING_PORT + self.kv_mgr.kv_args.engine_rank)
-        )
         self.decode_ip = self.kv_mgr.get_localhost()
         self.session_id = self.kv_mgr.get_session_id()
         self.kv_mgr.set_status(bootstrap_room, KVPoll.WaitingForInput)
+        self.bootstrap_addr = bootstrap_addr
+        self.prefill_engine_rank = None
+        self.prefill_tp_size = None
+        self.decode_port = KVRECEIVER_POLLING_PORT
+        self.zmq_port = 0
+
+    @cache
+    def _connect_bootstrap(self, endpoint: str):
+        socket = zmq.Context().socket(zmq.DEALER)
+        self.identity = str(uuid.uuid4()).encode()
```

Comment:
It is better to let ZMQ library to generate an identity rather than explicitly setting it